README.md

```diff
@@ -23,7 +26,3 @@ The short version:
 
 These steps are explained in more detail in the [Contributor
 guidelines](https://github.com/eclipse/rdf4j/blob/master/.github/CONTRIBUTING.md).
-
-## Release
-
-Run `./release.sh` and follow the prompts to create a release.
```

Comment:
We should have this info somewhere else? Maybe here: https://rdf4j.org/documentation/developer/#_developer_workflow_and_project_management ?

Reply:
Yeah good point. I took it out because it is only relevant to committers, not to contributors. But a link to our internal developer documents would be good to put in. I'll add something!